webpack/assets/javascripts/react_app/components/notifications/notifications.fixtures.js

```diff
@@ -0,0 +1,121 @@
+import immutable from 'seamless-immutable';
+
+export const emptyState = immutable({
+  notifications: {}
+});
+
+export const stateWithoutNotifications = immutable({
+  notifications: {
+    expandedGroup: 'React devs2',
+    isDrawerOpen: true
+  }
+});
+
+export const stateWithNotifications = immutable({
+  notifications: {
+    expandedGroup: 'React devs2',
+    isDrawerOpen: true,
+    notifications: {
+      '1': {
+        id: 1,
+        seen: true,
+        level: 'info',
+        text: null,
+        'created_at': '2017-02-23T05:00:28.715-05:00',
+        group: 'React devs',
+        actions: {}
+      },
+      '6': {
+        id: 6,
+        seen: true,
+        level: 'info',
+        text: 'Hi! This is a notification message',
+        'created_at': '2017-03-14T11:25:07.138-04:00',
+        group: 'React devs2',
+        actions: {
+          links: [
+            {
+              href: 'https://theforeman.org/blog',
+              title: 'Link to blog'
```

Comment:
This is fixture file, used for tests. Since there is no test currently about clicking the link,adding external property to the fixture doesnt matter